src/extension/src/azure/utils/nameGenerator.ts

```diff
@@ -52,11 +52,18 @@ export namespace NameGenerator {
     const fullDate = new Date(unixTimestamp);
     const year = fullDate.getFullYear().toString();
     // getMonth() returns month as a zero-based value
-    const month = (fullDate.getMonth() + 1).toString();
-    const date = fullDate.getDate().toString();
-    const hour = fullDate.getHours().toString();
-    const min = fullDate.getMinutes().toString();
-    const sec = fullDate.getSeconds().toString();
+    const month = padStart(fullDate.getMonth() + 1);
+    const date = padStart(fullDate.getDate());
+    const hour = padStart(fullDate.getHours());
+    const min = padStart(fullDate.getMinutes());
+    const sec = padStart(fullDate.getSeconds());
     return year.concat(month, date, hour, min, sec);
   }
+
+  function padStart(x: number): string {
```

Comment:
Doesn't a function for padStart already exist in ES? Although it's for strings so you'd have more .toStrings() haha. Your choice if you want to use it or not

Reply:
Yeap, I'm aware about that, but didn't want to do so many toString() just as you mentioned haha

Reply:
Oh you could also get rid of the function block then and just replace it with a simple x.toString().padStart if you want. But If not let me know it looks good otherwise.

Reply:
oh yea, that's a good idea